legacy/ui/legacy/src/main/java/com/fsck/k9/activity/MessageList.kt

```diff
@@ -579,6 +582,16 @@ open class MessageList :
             return
         }
 
+        featureFlagProvider.provide(FeatureFlagKey("material3_navigation_drawer"))
```

Comment:
Side note: It feels like `FeatureFlagKey` is unnecessary. We're never creating an instance that is passed around. Instead, whenever we interact with `FeatureFlagProvider`, we're using a string literal that has to be turned into a `FeatureFlagKey`.